app/src/main/java/com/example/pygmyhippo/user/EventFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.example.pygmyhippo.user;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.example.pygmyhippo.R;
+import com.example.pygmyhippo.common.Event;
+
+/**
+ * This fragment will hold a single events
+ * @author Katharine
+ * @version 1.0
+ * No returns and no parameters
+ */
+public class EventFragment extends Fragment {
+
+    // populate single event page with event information
+    private Event event;
+
+    public EventFragment() {
+    }
+
+    // this might be useful
```

Comment:
what is this for?

Reply:
i assume information about the event will be passed (to populate the page with info), everything now is hardcoded, but things will be set when i make the event